libminifi/include/core/ProcessSession.h

```diff
@@ -111,6 +111,7 @@ class ProcessSession : public ReferenceContainer {
   // import from the data source.
   void import(std::string source, const std::shared_ptr<core::FlowFile> &flow, bool keepSource = true, uint64_t offset = 0);
   void import(std::string source, std::vector<std::shared_ptr<FlowFileRecord>> &flows, bool keepSource, uint64_t offset, char inputDelimiter);
```

Comment:
If I remember well this is only used by tailfile.

Removing would be a breaking change, but I guess deprecation is ok as it becomes unreferenced.

Reply:
I agree that it should deprecated, but I am not sure what you specifically mean by that. Do we have a process for marking functions deprecated?

Reply:
I have added the DEPRECATED macro to MiNiFi.